tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/resource/PetResource.java

```diff
@@ -303,6 +315,7 @@ public Response addPet(Pet pet) {
     public Response updatePet(
         @Parameter(
             name ="petAttribute",
+            in = ParameterIn.QUERY,
```

Comment:
Should this be a `@Parameter` at all?  The `Pet` argument is presumably the request body for this operation.